rotkehlchen/exchanges/coinbase.py

```diff
@@ -898,16 +899,15 @@ def _deserialize_asset_movement(self, raw_data: dict[str, Any]) -> AssetMovement
 
             # Only get address/transaction id for "send" type of transactions
             address = None
-            transaction_id = None
-            transaction_url = raw_data.get('id', '')
+            transaction_id = raw_data.get('id')
+            transaction_url = None
```

Comment:
you are switching them from before. This changes the implementation. your change appears to be correct but that still means the implementation changes.

You understand this means that after this PR half your asset movements will have swapped transactionid/url for coinbase for things before running this version.

So if this is the right approach this needs to also be swapped during the DB upgrade. Right?

Reply:
These were being used to set the `transaction_id` and `link` properties of the AssetMovement, neither of which exist in the new AssetMovement. Now `transaction_id` and `transaction_url` are simply set in `extra_data`, as well as `transaction_id` being used for the `unique_id` (to create the `event_identifier`).

As discussed with yabir in [discord](https://discord.com/channels/657906918408585217/1080783409640648765/1316794835830706207), the `link` (which is what may contain this `id` value) will be added to `extra_data`. So while it may not have the same key name, it should be present in the upgraded events' `extra_data`.